ihmc-robotics-toolkit/src/test/java/us/ihmc/robotics/math/frames/YoMatrixTest.java

```diff
@@ -25,6 +25,8 @@ public void testSimpleYoMatrixExample()
       assertEquals(maxNumberOfColumns, yoMatrix.getNumCols());
 
       DMatrixRMaj denseMatrix = new DMatrixRMaj(maxNumberOfRows, maxNumberOfColumns);
+      yoMatrix.reshape(maxNumberOfRows, maxNumberOfColumns);
+      yoMatrix.zero();
```

Comment:
The zero method got changed a few months back, so this has to be done instead to get the "out of poitns" to be Nan

Reply:
`yoMatrix.reshape(maxNumberOfRows, maxNumberOfColumns);` seems unnecessary for the test to pass.

Reply:
Did you check the test out locally and verify that it isn't necessary, or is that your guess from looking at it?

Reply:
Tested locally.